src/lib/Microsoft.Health.Fhir.Ingest.Common/Template/FhirCodeableConcept.cs

```diff
@@ -4,14 +4,17 @@
 // -------------------------------------------------------------------------------------------------
 
 using System.Collections.Generic;
+using Newtonsoft.Json;
 
 namespace Microsoft.Health.Fhir.Ingest.Template
 {
     public class FhirCodeableConcept
     {
+        [JsonProperty(Required = Required.Always)]
```

Comment:
It would actually still be a valid Observation if Text was null. Does the Function use this property for something that would make this required?

Reply:
Yes this isn't required.

Reply:
Thank you both for pointing out. I'll remove this requirement accordingly.